File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.py
class Solution(object):
    def missingMultiple(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        n = 1
        while(n * k in nums):
            n += 1
        return n * k
        

File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.java
class Solution {
    public int missingMultiple(int[] nums, int k) {
        HashSet<Integer> hs = new HashSet<>();
        for(int el : nums)  hs.add(el);
        int n = 1;
        while(hs.contains(n *k)){
            n++;
        }
        return n * k;
    }
}